src/http-fetcher.js

```diff
@@ -1,5 +1,5 @@
 export default function httpFetcher(url, options = {}) {
-  return function fetcher(graphQLParams) {
+  return function fetcher(graphQLParams, headers = null) {
```

Comment:
headers don't need a default value of null since they'll be undefined by default anyway.

Reply:
Wouldn't `...headers` below error then since it's not defined?

Reply:
Ah you meant `fetcher(graphQLParams, headers)` 👍

I was thinking leaving out `headers` entirely for some reason which wouldn't work

Reply:
yus. undefined value. not undefined name 😄